agent/util-scripts/pbench-tool-meister

```diff
@@ -1152,8 +1153,16 @@ def main(argv):
             #   a. handle graceful termination (TERM, INT, QUIT)
             #   b. log operational state (HUP maybe?)
 
+            #ADDING METADATA GRAB HERE
+            meta_raw = redis_server.get("tool-metadata")
+            if params_raw is None:
+                logger.error('Metadata was never loaded.')
+            meta_str = meta_raw.decode("utf-8")
+            tool_metadata = json.loads(meta_str)
+            logger.info(f"Metadata: {tool_metadata}")
```

Comment:
I'm not sure this is the only place we'll want access to the metadata. Maybe something like a ToolMetadata object that can be constructed either from the JSON or from the Redis document, and passed around as context with methods to abstract the representation?

Reply:
Both the on-disk JSON document and the in-Redis JSON document should be the same, so an entity constructing the `ToolMetadata` object can just read from on or the other to then construct the object.